src/compiler/program.ts

```diff
@@ -226,6 +226,22 @@ namespace ts {
         for (const diagnostic of diagnostics) {
             output += formatDiagnostic(diagnostic, host);
         }
+

```

Comment:
No.. This is not right. 
You want to change https://github.com/Microsoft/TypeScript/blob/master/src/tsc/tsc.ts#L228 to 
```ts
const exitStatus = emitFilesAndReportErrors(program, reportDiagnostic, s => sys.write(s + sys.newLine), shouldBePretty(options) ? numberOfErrors => {
            const d = createCompilerDiagnostic(numberOfErrors === 1 ? Diagnostics.Found_1_error : Diagnostics.Found_0_errors)
            sys.write(`${sys.newLine}${flattenDiagnosticMessageText(d.messageText, sys.newLine)}${sys.newLine}${sys.newLine}`);
        }: undefined);
        
```

Reply:
Got it, thanks!

Reply:
Okay, I've made this change in the latest commit to this PR. Thank you for the guidance.

Since this code now only prints the summary in pretty mode, the diff of this PR is a lot smaller. It seems like that's the direction people want to go in, which makes sense (especially as it's the default now). Should we add a test for this? Let me know, and thanks again 👍

Reply:
To test this it needs larger change. Let me take this up instead. Thanks.

Reply:
Okay, thanks!